src/layout/header/GitHubLink/GitHubLink.scss

```diff
@@ -2,11 +2,11 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  font-size: 1.5em; 
+  font-size: 1.5em;
   cursor: pointer;
-
+  transition: all 0.1s;
```

Comment:
question: why do we need this transition rule?

Reply:
I just wanted to add a bit of flavor when you hover over the github logo or the bulb.